src/transactions/queries/ReceivedFioRequests.ts

```diff
@@ -36,7 +38,7 @@ export class ReceivedFioRequests extends Query<ReceivedFioRequestsResponse> {
           value.content = content
           requests.push(value)
         } catch (e) {
-          //
+          if (this.includeEncrypted) { requests.push(value) }
```

Comment:
remove `{` `}`

Reply:
ok, but { } is required in the tslint rules